Banyan/src/jobs.jl

```diff
@@ -199,8 +199,45 @@ function get_jobs(cluster_name=nothing; status=nothing, kwargs...)
     if !isnothing(status)
         filters["status"] = status
     end
-    response =
-        send_request_get_response(:describe_jobs, Dict{String,Any}("filters"=>filters))
+    
+    # finished = false
+    # Jobs = []
+    # Last_eval, next_jobs = describe_jobs()
+    # jobs.append(next_jobs)
+    # If (last_eval = None)
+    # Finished = True
+    # While finished = False0
+    # New_eval, next_jobs = describe_jobs(last_eval)
+    # jobs.append(next_jobs)
+    # if(new_eval = None)
+    #         Finished = True
+    # Last_eval = new_eval
+    # return jobs
+
+    
+    response = Dict("last_eval_key" => 50394, "jobs" => [])
+    finished = false
+    indiv_response = send_request_get_response(:describe_jobs, Dict{String,Any}("filters"=>filters))
+    response = indiv_response
+    if indiv_response["last_eval"] == nothing 
+        finished = true
+    else
+        curr_last_eval = indiv_response["last_eval"]
+        while finished == false
+            println(curr_last_eval)
```

Comment:
Maybe we can put this in an `if is_debug()` if statement, so that we only print this out if we are in debug mode.